contrib/gcppubsub/cmd/receive_adapter/main.go

```diff
@@ -64,11 +58,17 @@ func main() {
 		log.Fatalf("Unable to create logger: %v", err)
 	}
 
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		log.Fatal("Failed to process env var", zap.Error(err))
```

Comment:
Is this a sugared logger? If so, it should be `Fatalw` (my preference is just switch to a desugared logger to make sure these bugs don't occur).